### PR TITLE
Custom items 12/10/22

### PR DESCRIPTION
**Pull request by Alberyk**
Adds the following custom items:
https://forums.aurorastation.org/topic/17397-ahekes-thermal-coat
https://forums.aurorastation.org/topic/17432-the-ahzi-fleet-kit
https://forums.aurorastation.org/topic/17562-ielias-starfinder

Also removes items from characters that no longer exists or have not been played for over a year

### CODE REVIEW

File: code/modules/customitems/item_defines.dm [hunk 1]
```diff
@@ -2286,3 +1962,128 @@ All custom items with worn sprites must follow the contained sprite system: http
 	name = "blue taffy"
 	icon_state = "blue_taffy"
 	reagent_data = list(/decl/reagent/nutriment = list("salty-sweet, tangy taffy" = 1))
+
+
+/obj/item/clothing/suit/storage/fluff/aheke_coat //Hengsha Thermal Coat - Aheke Han'san - hawkington
+	name = "hengsha thermal coat"
+	desc = "A very heavy, well padded and generally quite large grey overcoat, probably made for an Unathi, made out of a rather stiff, almost leathery type substance. With a brown synthetic fur \
```
Comment (SleepyGemmy) [inline code/modules/customitems/item_defines.dm:1969]:
```suggestion
	desc = "A very heavy, well padded, and generally quite large grey overcoat, probably made for an Unathi, made out of a rather stiff, almost leathery type substance. With a brown synthetic fur \
```

File: code/modules/customitems/item_defines.dm [hunk 2]
```diff
@@ -2286,3 +1962,128 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+	collar and even more fleecing inside. Interestingly enough there does not seem to be an easy way of closing it. Emblazoned on the back in a large red letters is \"K6\""
```
Comment (SleepyGemmy) [inline code/modules/customitems/item_defines.dm:1970]:
```suggestion
	collar and even more fleecing inside. Interestingly enough, there does not seem to be an easy way of closing it. Emblazoned on the back in large red letters is \"K6\"."
```

File: code/modules/customitems/item_defines.dm [hunk 3]
```diff
@@ -2286,3 +1962,128 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+	icon = 'icons/obj/custom_items/aheke_coat.dmi'
+	icon_override = 'icons/obj/custom_items/aheke_coat.dmi'
+	icon_state = "aheke_coat"
+	item_state = "aheke_coat"
+	contained_sprite = TRUE
+
+
+/obj/structure/sign/flag/unathi_fleet
+	name = "unathi fleet flag"
+	desc = "A flag bearing the easily recognizable inconography of Unathi fleets, this one depicting a Sinta slain by spears under an omniscient, uncaring eye."
```
Comment (SleepyGemmy) [inline code/modules/customitems/item_defines.dm:1980]:
```suggestion
	desc = "A flag bearing the easily recognizable iconography of the Unathi fleets, this one depicting a Sinta slain by spears under an omniscient, uncaring eye."
```

File: code/modules/customitems/item_defines.dm [hunk 4]
```diff
@@ -2286,3 +1962,128 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+
+/obj/structure/sign/flag/unathi_fleet/left
+	icon_state = "unathi_fleet_l"
+
+/obj/structure/sign/flag/unathi_fleet/right
+	icon_state = "unathi_fleet_r"
+
+/obj/item/flag/fluff/ahzi_flag //Unathi Fleet Flag - Ankala Ahzi - captaingecko
+	name = "unathi fleet flag"
+	desc = "A flag bearing the easily recognizable inconography of Unathi fleets, this one depicting a Sinta slain by spears under an omniscient, uncaring eye."
```
Comment (SleepyGemmy) [inline code/modules/customitems/item_defines.dm:1991]:
```suggestion
	desc = "A flag bearing the easily recognizable iconography of the Unathi fleets, this one depicting a Sinta slain by spears under an omniscient, uncaring eye."
```

File: code/modules/customitems/item_defines.dm [hunk 5]
```diff
@@ -2286,3 +1962,128 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+	icon_override = 'icons/obj/custom_items/ahzi_items.dmi'
+	icon_state = "ahzi_flag"
+	flag_path = "unathi_fleet"
+
+/obj/item/flag/fluff/ahzi_flag/l
+	flag_size = 1
+
+/obj/item/clothing/accessory/armband/fluff/ahzi_armband //Unathi Fleet Armband - Ankala Ahzi - captaingecko
+	name = "unathi fleet armband"
+	desc = "An armband bearing the easily recognizable inconography of Unathi fleets, this one depicting a Sinta slain by spears under an omniscient, uncaring eye."
```
Comment (SleepyGemmy) [inline code/modules/customitems/item_defines.dm:2002]:
```suggestion
	desc = "An armband bearing the easily recognizable iconography of the Unathi fleets, this one depicting a Sinta slain by spears under an omniscient, uncaring eye."
```

File: code/modules/customitems/item_defines.dm [hunk 6]
```diff
@@ -2286,3 +1962,128 @@ All custom items with worn sprites must follow the contained sprite system: http
...
+	icon = 'icons/obj/custom_items/ahzi_items.dmi'
+	icon_override = 'icons/obj/custom_items/ahzi_items.dmi'
+	icon_state = "ahzi_armband"
+	item_state = "ahzi_armband"
+	contained_sprite = TRUE
+
+
+/obj/item/deck/tarot/nralakk/fluff/ielia_tarot //Starfinder - Ielia Aliori-Quis'Naala - shestrying
+	name = "starfinder"
+	desc = "A small, bronze ball. It is heavy in the hand, and seems to have no switches or buttons on it. "
```
Comment (SleepyGemmy) [inline code/modules/customitems/item_defines.dm:2012]:
```suggestion
	desc = "A small, bronze ball. It is heavy in the hand and seems to have no switches or buttons on it. "
```

**Review by SleepyGemmy — CHANGES_REQUESTED** [review]
some sprites in "icons/obj/custom_items/ielia_tarot.dmi" also have uppercase names and spaces instead of underscores.

**Review by SleepyGemmy — APPROVED** [review]
requested changes addressed.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**